modules/importer.js

```diff
@@ -22,10 +22,24 @@ export default class Importer {
 	}
 
 	parseCsv(buffer) {
-		const data = buffer.toString().split(/\r\n|\r|\n/);
+		let data = buffer.toString().split(/\r\n|\r|\n/);
 
 		while (!data[data.length - 1]) data.pop();
 
-		return data.map(string => string.split(","));
+		data = data.map(string => string.split(","));
+		const keys = data.shift();
```

Comment:
`const keys = data[0]`
omit such operations like `shift(), unshift()`, because they decrease performance of functions

Reply:
fixed